app/app.css

```diff
@@ -7,3 +7,10 @@ table thead tr th{text-align:center}
 .playerList{margin:5rem 40rem 3rem 40rem}
 
 .errorLength{color:red}
+
```

Comment:
You didn't notice that this file is automatically created by `gulp`. You should your css in another place.
`SMACSS` approach has been followed and all css rules are generated using `*.scss` files.